controllers/modelmesh/runtime.go

```diff
@@ -342,16 +342,15 @@ func (m *Deployment) addPassThroughPodFieldsToDeployment(deployment *appsv1.Depl
 	rts := m.SRSpec
 	// these fields map directly to pod spec fields
 	// supported architectures are "amd64" and "arm64", "ppc64le"
-	// and "s390x" are not supported by tensorflow
-	// (https://github.com/kserve/modelmesh-runtime-adapter/pull/38#discussion_r1156749259)
+	// and "s390x" 
```

Comment:
This comment needs some revision. It seems like the original comment says that both `pp64le` and `s390x` are not supported. The new comment is a bit confusing and looks like they are now both supported by this change. Can you please confirm and revise?

Reply:
Tensorflow support for s390x and pp64cle is still required & i am currently working on it. Here i have added s390x as part of node affinity. If it is still confusing , do let me know i will update the comment

Reply:
Thanks for the info, but the comment still needs to be corrected either way. The change you made results in the comment:
```
supported architectures are "amd64" and "arm64", "ppc64le" and "s390x"
```

If all four are supported then it should be:
```
supported architectures are "amd64", "arm64", "ppc64le" and "s390x"
```

But it sounds like they're not, so maybe it's supposed to be something like the following with updated information on the unsupported arch:
```
supported architectures are "amd64", "arm64", and "s390x". "ppc64le" is not yet supported by tensorflow (https://github.com/kserve/modelmesh-runtime-adapter/pull/38#discussion_r1156749259).
```

Reply:
Updated comments in latest fix